### PR TITLE
email updates

### PR DESCRIPTION
**Pull request by laijasmine**
- added information on data portals
- added request for contact for large submissions
- shortened where possible to focus on points: what we are here for and how we can help

### CODE REVIEW

**Comment by laijasmine** [comment]
@aebudden and @erinlynmclean

File: inst/emails/contact_annual_report [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+Prior to issuing a DOI, there is a processing period for the Arctic Data Centre to review a submission. While we strive to process data as quickly as possible, larger data sets can take significantly more time to process. We advise that this is taken to account when planning a submission to the Arctic Data Center for your report.
 
- If you have any questions about this process please reply to this email and we will respond as soon as possible.
+If you have any questions about any of the above, don't hestitate to email us and we will respond as soon as possible.
```
Comment (mbjones) [inline inst/emails/contact_annual_report]: I think the reason the prior text said "respond to this email"  rather than "email us" is to keep their response tied to the RT ticket created for their award.  If you still want to change it to "email us", we should provide the `support@arcticdata.io` email address.

File: inst/emails/contact_aon_recurring [hunk 2]
```diff
@@ -1,12 +1,12 @@
 Dear %s,
 
-The NSF-funded Arctic Data Center (https://arcticdata.io/) support team would like to remind you that as part of your NSF Arctic Observing Network (AON) award you are required to submit newly collected data every six months. If you have not already done so please submit any new data to your NSF-approved repository.
+The NSF-funded Arctic Data Center support team would like to remind you that as part of your NSF Arctic Observing Network (AON) award you are required to submit newly collected data every six months. If you have not already done so please submit any new data to your NSF-approved repository.
 
-If you have not already done so, please submit your applicable data and metadata by going to https://arcticdata.io/catalog/#share. You can also visit our data submission guidelines page (https://arcticdata.io/submit/) for more information about the Arctic Data Center submission process.
+Submissions to the Arctic Data Centre can be done here: https://arcticdata.io/catalog/#share. You can also visit our data submission guidelines page (https://arcticdata.io/submit/) for more information about the Arctic Data Center submission process.
```
Comment (mbjones) [inline inst/emails/contact_aon_recurring]: "here' links are hard to follow. Better to use a more descriptive phrase, like "can be done via our online web form: https..."

File: inst/emails/contact_aon_recurring [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
-Please be sure to submit your data and metadata to the Arctic Data Center sufficiently before your report is due to allow adequate processing time for us to review your submission and for any needed revisions to be prepared.  While we strive to process data as quickly as possible, larger data sets can take significantly more time to process, so please plan accordingly.  Also, there will be less need for multiple rounds of revisions if your initial submission uses open, non-proprietary data formats, includes data that is well-organized, and includes high quality, detailed documentation.  For example, you should document all aspects of your data, including detailed methodological descriptions and structural information about the contents of your data.  See our data submission guidelines page (https://arcticdata.io/submit/) for recommendations.
+Prior to issuing a DOI, there is a processing period for the Arctic Data Centre to review a submission. While we strive to process data as quickly as possible, larger data sets can take significantly more time to process. Please take this into account when planning a submission for your report.
```
Comment (mbjones) [inline inst/emails/contact_aon_recurring]: Some of the detail in this paragraph is useful.  At a minimum, I think we should retain the "See our data submission guidelines page (https://arcticdata.io/submit/) for recommendations." Why was this paragraph removed?

File: inst/emails/contact_initial_ans [hunk 4]
```diff
@@ -2,15 +2,19 @@ Dear %s,
...
+With these requirements in mind, we strongly recommend that you plan to keep detailed and well-documented data and metadata records throughout your project. Some guidelines are provided here: https://arcticdata.io/submit/#organizing-your-data. If you are interested, we also have annual training opportunities posted here: https://arcticdata.io/training/.
 
-If you have any questions about the Arctic Data Center or NSF requirements, please reply to this email and we will respond as soon as possible. We will also contact you again near the end of your award with reminders about the data submission requirements.
+If your research will involve large data submissions (a total size of 500MB or more), please get in touch with us so we can create a plan together for those datasets.
```
Comment (mbjones) [inline inst/emails/contact_initial_ans]: I think this should be a higher limit. Maybe > 100GB total size, or more than 500 files. They should be able to handle a few hundred GB uploads through the web form with ease. What was the reason for making the limit so low?

File: inst/emails/contact_initial_ans [hunk 5]
```diff
@@ -2,15 +2,19 @@ Dear %s,
...
+
+We also wanted to highlight the data portals feature at the Arctic Data Center. If your project involves multiple related datasets, such as those that are part of a larger research project or organization, a data portal can allow datasets to be discovered together. More information on how to create a data portal and examples can be found here: https://arcticdata.io/data-portals/.
```
Comment (mbjones) [inline inst/emails/contact_initial_ans]: This is good, but we should add more from our promotional language describing the data portals feature, and give a couple of examples (e.g., a portal for a single grant, a portal for multiple groups collaborating across time on multiple grants, and thematic portals on specific topics (e.g., Salmon) or locations (e.g., Summit Station). Maybe borrow some language from the data-portals page.

File: inst/emails/contact_initial_aon [hunk 6]
```diff
@@ -2,20 +2,25 @@ Dear %s,
...
-
-With these requirements in mind, we strongly recommend that you plan to keep detailed and well-documented data and metadata records throughout your project. Typically, NSF researchers already do a great job of collecting data and metadata, but you will want to ensure your data and metadata are collected in a manner that can be easily accessed and understood by other researchers after it is published. We recommend that you visit our data submission guidelines page (https://arcticdata.io/submit/) to get a better understanding of what is expected for submissions to the Arctic Data Center.
-
-Please note that for AON program projects, NSF requires that:
-    - Real-time data be made publicly available immediately. If there is any question about what constitutes real-time data, please contact the appropriate NSF program officer.
+Please note that for AON program projects, NSF has the additional requirements:
+    - Real-time data be made publicly available immediately.
```
Comment (mbjones) [inline inst/emails/contact_initial_aon]: "If there is any question about what constitutes real-time data, please contact the appropriate NSF program officer." This was in there for a reason. The language can be softened, but they need to talk to program officers about what part of their data is real-time.

File: inst/emails/contact_initial_aon [hunk 7]
```diff
@@ -2,20 +2,25 @@ Dear %s,
...
     - All data sets and derived data products be accompanied by a metadata profile and full documentation that allows the data to be properly interpreted and used by other researchers.
 
-If you have any questions about the Arctic Data Center or NSF requirements, please reply to this email and we will respond as soon as possible. We will also contact you again near the end of your award with reminders about the data submission requirements.
+More detailed information about what will be expected of you can be found at:
+https://arcticdata.io/submit/#who-must-submit.
+
+With these requirements in mind, we strongly recommend that you plan to keep detailed and well-documented data and metadata records throughout your project. Some guidelines are provided here: https://arcticdata.io/submit/#organizing-your-data for best practices. For those who are interested, we also have annual training opportunities posted here: https://arcticdata.io/training/.
+
+If your research will involve large submissions of data (a total size of 500MB or more), please get in touch with us so we can create a plan together for those datasets.
```
Comment (mbjones) [inline inst/emails/contact_initial_aon]: Same issue as above.

Comment (mbjones) [reply inst/emails/contact_initial_aon]: Also, note that AON data is often semi-automated from instruments, and that it may be worthwhile for them to use our API to deposit data files from a script automatically as it is generated and/or processed.  We can work with them on how to use the API to automate their processes. I think we should mention this in here somewhere.

File: inst/emails/contact_one_month_remaining [hunk 8]
```diff
@@ -4,13 +4,13 @@ The NSF-funded Arctic Data Center (https://arcticdata.io/) support team would li
...
+If you have not already done so, please submit your applicable data and metadata by going to https://arcticdata.io/catalog/#share. You can also visit our data submission guidelines page (https://arcticdata.io/submit/) for more information about the Arctic Data Center submission process.
 
-See the reporting guidelines{https://www.nsf.gov/pubs/2016/nsf16055/nsf16055.jsp} from NSF for details, but generally NSF requires that URL's for archived metadata and data should be included in these reports in the section entitled "Products-Websites.".
+Prior to issuing a DOI, there is a processing period for the Arctic Data Centre to review a submission. While we strive to process data as quickly as possible, larger data sets can take significantly more time to process. We advise that this is taken to account when planning a submission to the Arctic Data Center for your report.
```
Comment (mbjones) [inline inst/emails/contact_one_month_remaining]: Do we want to mention that, while many data sets can be processed in a day or two, larger or more complex data can sometimes take a couple of weeks to process?  I think giving people an idea of the time scale would be helpful in this and other emails above.

**Review by mbjones — COMMENTED** [review]
These revisions look good, @laijasmine -- suggestions inline below.  I thin the portals feature could use a bit more "sales" orientation.  And I've made some suggestions on some details that should probably be added back into the emails.

File: inst/emails/contact_aon_recurring (hunk 2 above)
Comment (laurenwalker) [reply inst/emails/contact_aon_recurring]: The `https://arcticdata.io/catalog/#share` URL still works, but is old. We should use `https://arcticdata.io/catalog/submit`

Comment (laurenwalker) [reply inst/emails/contact_aon_recurring]: Also, fix `Centre` typo 🙂

File: inst/emails/contact_initial_ans (hunk 4 above)
Comment (laijasmine) [reply inst/emails/contact_initial_ans]: Changed! I think this was a typo from a message I was referring to

File: inst/emails/contact_initial_ans (hunk 5 above)
Comment (laijasmine) [reply inst/emails/contact_initial_ans]: there were more examples but I opted for less text because the emails were approaching 2 pages and I was worried about the recipient tuning out most of it if we came with a wall of text. I'll play around with the text to see if I can get a balance of both

File: inst/emails/contact_aon_recurring (hunk 2 above)
Comment (aebudden) [reply inst/emails/contact_aon_recurring]: > "here' links are hard to follow. Better to use a more descriptive phrase, like "can be done via our online web form: https..."

Applies throughout - e.g. to the portals page link

**Comment by aebudden** [comment]
For the 'up-sell' of the portals maybe include language such as: 
We also wanted to highlight the data portals feature at the Arctic Data Center. Portals are dedicated sites that allow individuals and teams to create a collection of related datasets. They facilitate discovery, have collection specific metrics, can include custom search features and can be branded to reflect your organization of collaboration. For more information and tutorials on how to get started, visit our data portal page at https://arcticdata.io/data-portals/.

The *can*  before 'include custom features' is optional. I prefer the sentence without however this isn't something users can do simply for themselves so I didn't want to misrepresent.

**Comment by erinlynmclean** [comment]
I promise I'm not ignoring this thread...how do I see the whole text of the emails at once?

**Comment by mbjones** [comment]
@erinlynmclean you can see the full set of files in the pull request under the 'View file' menu item in GitHub. If you want them locally, you can check out the pull request branch. If you want to browse them on the web, they are all here: https://github.com/NCEAS/awards-bot/tree/3ad634960cd4bb72886189cba9927c83e29d2149/inst/emails

**Comment by erinlynmclean** [comment]
My edits:

**contact_annual_report**:
- Line 5 should be URLs not URL's
- Line 9 should have Center not Centre; datasets is one word not two
- Line 11 hesitate is spelled incorrectly

**contact_aon_recurring**:
- Line 3 should have a comma after "already done so"
- Line 7 should have Center not Centre; datasets is one word not two
- Line 9 hesitate is spelled incorrectly

**contact_initial_ans**: everything looks good, and I like Amber's way of upselling the portals.

**contact_initial_aon**:
- Line 12 datasets is one word not two
- Line 23 hesitate is spelled incorrectly

**contact_initial_social_sciences**:
- Line 11 should say a metadata record or metadata (no a); sensitive spelled incorrectly
- Line 19 hesitate is spelled incorrectly

**contact_one_month_remaining**:
- Line 3 should be less than one month until the end date of your NSF award %s %s
- Line 7 should be URLs not URL's; extra period after websites (should be after the quotation mark, not before)
- Line 11 should have Center not Centre; remove a before "small datasets";  add comma after "small datasets"

**Comment by laijasmine** [comment]
@mbjones @erinlynmclean @aebudden 

Thank you everyone for their edits! I would love another look when you have a chance to finalize the changes before merging!

**Review by erinlynmclean — APPROVED** [review]
These all look good to me!
